composer.json

```diff
@@ -26,10 +29,14 @@
     },
     "require": {
         "php": ">=7.1",
+        "yiisoft/yii2": "~2.0.13",
         "yiisoft/yii2-httpclient": "^2.0"
     },
     "require-dev": {
-        "codeception/codeception": "^3.0",
+        "codeception/codeception": "^4.0",
+        "codeception/module-asserts": "^1.0.0",
+        "codeception/module-yii2": "^1.0.0",
+        "dealerdirect/phpcodesniffer-composer-installer": "^0.7.1",
```

Comment:
why do we need it? yii2-coding-standards comes with phpcs as dependency

Reply:
that way everyones IDE will be very happy when just saying Yii2 code standard. even your docker image that has phpcs installed globally